frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.unit.spec.tsx

```diff
@@ -13,15 +13,15 @@ const setup = ({
   isAdmin = false,
   hasPublicLink = false,
   isResourcePublished = false,
-  isApplicationEmbeddingEnabled = false,
+  isStaticEmbeddingEnabled = false,
```

Comment:
just a rename to make it make more sense. I'm not sure what _application embedding_ is